llarp/ev/ev_libuv.cpp

```diff
@@ -474,13 +474,19 @@ namespace libuv
       if (self == nullptr)
         return -1;
       auto buf = uv_buf_init((char*)ptr, sz);
-      return uv_udp_try_send(&self->m_Handle, &buf, 1, to);
+      auto ret =
+          uv_udp_try_send(&self->m_Handle, &buf, 1, (sockaddr*)static_cast<const sockaddr_in*>(to));
+      if (ret < 0)
+      {
+        llarp::LogError("udp sendto failed: ", uv_strerror(ret));
+      }
+      return ret;
     }
 
     bool
     Bind()
     {
-      auto ret = uv_udp_bind(&m_Handle, m_Addr, 0);
+      auto ret = uv_udp_bind(&m_Handle, (sockaddr*)static_cast<const sockaddr_in*>(m_Addr), 0);
```

Comment:
Is...is that a C-cast of a static_cast?  That feels wrong...

Reply:
it is very wrong

Reply:
this is better than the old implicit conversions done before, this is more explicit about what is going on.